code/bit_manipulation/src/addition_using_bits/addition_using_bits.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+
+// Part of Cosmos by OpenGenus Foundation
+int bitwiseAddition(int n, int m)
+{
+    while (m != 0)
+    {
+        int i = n & m;  // Variable i keeps track of bits that carry over
```

Comment:
This variable should be declared outside the loop.

Reply:
@abdouskamel Why?

Reply:
@Dhenderson10 Could you rename i to carry?